### PR TITLE
Npm push

### PR DESCRIPTION
**Pull request by Meyanis95**
Version 0.0.3 of the package published
- Web prover working 🎉

### CODE REVIEW

File: .github/workflows/01-workflow.yaml [hunk 1]
```diff
@@ -9,6 +9,10 @@ on:
       - main
 
 jobs:
+  env:
```
Comment (vuvoth) [inline .github/workflows/01-workflow.yaml:12]: env should be in "root" property i think 
```yaml
env:
   ... 
jobs: 
  ...
```

Comment (Meyanis95) [reply .github/workflows/01-workflow.yaml]: Thanks :)

**Comment by Meyanis95** [comment]
It seems that's it's not an env variables problem but that the web prover test is not working in GitHub actions.
I'll check how to solve this problem, but maybe linked to the size of the zkey file.

**Comment by vuvoth** [comment]
I think the problem is we fetch generate verify key every time we run CI. This mean the key pair(zkey, verifier key) not match

File: src/pcd.ts [hunk 2]
```diff
@@ -78,8 +79,20 @@ function getVerifyKey() {
...
 }
 
-export async function verify(pcd: IdentityPCD): Promise<boolean> {
-  const vk = getVerifyKey()
+export async function verify(
+  pcd: IdentityPCD,
+  isWebEnv?: boolean
+): Promise<boolean> {
+  let vk
+  if (isWebEnv === true) {
```
Comment (vuvoth) [inline src/pcd.ts:87]: I don't think we should fetch verify key from another URL.

Comment (Meyanis95) [reply src/pcd.ts:87]: How do you think we should handle that?

I was thinking that the web verifying and proving should rely on the same "state", and by so using the same artifacts stored on web.

Comment (Meyanis95) [reply src/pcd.ts:87]: What do you mean by mock server?

For now I'm using aws S3

Comment (vuvoth) [reply src/pcd.ts:87]: I mean a location server to store proving key and we can use key have same "state" with artifacts folder.

Comment (Meyanis95) [reply src/pcd.ts:87]: Yes that's a good idea!!
I added a ticket here => https://github.com/privacy-scaling-explorations/anon-aadhaar/issues/44

Can we validate this PR related to the web prover and add the mock server on the other ticket?

Comment (vuvoth) [reply src/pcd.ts:87]: I will approve PR after resolve conflict.

**Review by vuvoth — APPROVED** [review]

**Pull request merged**